### ISSUE TITLE
Error while analyzing golang codebase itself

### ISSUE THREAD

**Issue by tuxdna** [gofed/gofedlib#17]
Checkout go codebase:

```
git clone git@github.com:golang/go.git
```

Run gofedlib

```
$ ./gofedlib-cli -a /home/saleem/work/learn/go
Traceback (most recent call last):
  File "./gofedlib-cli", line 99, in <module>
    result['api'] = api(args.path)
  File "/home/saleem/work/learn/gofedlib/gofedlib/go/functions.py", line 5, in api
    return GoSymbolsExtractor(source_code_directory).extract().exportedApi()
  File "/home/saleem/work/learn/gofedlib/gofedlib/go/symbolsextractor/extractor.py", line 323, in extract
    raise ExtractionError(err_msg)
gofedlib.types.ExtractionError: directory /test/fixedbugs/issue5614.dir contains definition of more packages, i.e. rethinkgo
```

**Comment by tuxdna** [comment]
Even with trailing trailing whitespace on the input path:

```
$ gofedlib-cli --dependencies-main --dependencies-packages /home/saleem/work/learn/go/
Traceback (most recent call last):
  File "/home/saleem/rhat/bayesian/github/common/env2/bin/gofedlib-cli", line 109, in <module>
    result.update(get_dependencies(project_packages(args.path), selection))
  File "/home/saleem/rhat/bayesian/github/common/env2/lib/python2.7/site-packages/gofedlib/go/functions.py", line 8, in project_packages
    return GoSymbolsExtractor(source_code_directory).extract().packages()
  File "/home/saleem/rhat/bayesian/github/common/env2/lib/python2.7/site-packages/gofedlib/go/symbolsextractor/extractor.py", line 239, in extract
    cme.extract()
  File "/home/saleem/rhat/bayesian/github/common/env2/lib/python2.7/site-packages/gofedlib/go/contentmetadataextractor.py", line 159, in extract
    self._orNodes()
  File "/home/saleem/rhat/bayesian/github/common/env2/lib/python2.7/site-packages/gofedlib/go/contentmetadataextractor.py", line 53, in _orNodes
    _or = _or or self._orNodes("%s/%s" % (node, n))
  File "/home/saleem/rhat/bayesian/github/common/env2/lib/python2.7/site-packages/gofedlib/go/contentmetadataextractor.py", line 49, in _orNodes
    if self._nodes[node] == []:
KeyError: '/doc'
```

**Comment by tuxdna** [comment]
Fixed in - https://github.com/gofed/gofedlib/pull/24

**Comment by tuxdna** [comment]
Marking as closed via #24
